presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,452 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+* ``subGroups`` (optional): list of sub-groups.A list of sub-groups within the resource group.
+  Each sub-group can have its own set of properties.
 
-    * ``executionTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum time a query may take to execute.
+.. _scheduleweight-example:
 
-    * ``totalMemoryLimit`` (optional): Specify an absolute value (i.e. ``1GB``)
-       for the maximum distributed memory a query may consume.
+Scheduling Weight Example
+^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    * ``cpuTimeLimit`` (optional): Specify an absolute value (i.e. ``1h``)
-       for the maximum CPU time a query may use.
+Schedule weighting is a method of assigning a priority to a resource. Sub-groups
+with a higher scheduling weight are given higher priority. For example, to
+ensure timely execution of scheduled pipelines queries, weight them higher than
+adhoc queries. The scheduling weight in PrestoDB's Resource Groups is used when the parent group's
+scheduling policy is set to ``weighted``. It determines how resources are distributed
+among the subgroups. A higher weight means that the group gets a larger share of the parent group's resources.
 
-* ``workerPerQueryLimit`` (optional): specifies the minimum number of workers that have to
-  be available for each query. Intended to be used in elastic clusters where number of workers
-  varies over time.
+Here's an example:
 
-* ``subGroups`` (optional): list of sub-groups.
+Let's say we have a root resource group ``global`` with two subgroups: ``engineering`` and ``marketing``.
+The ``engineering`` subgroup has a scheduling weight of 3, and the ``marketing`` subgroup has a
+scheduling weight of 1. In this setup, the engineering subgroup will get 75% of the
+parent group's resources (because 3 is 75% of the total weight of 4), and the "marketing" subgroup will
+get 25% of the parent group's resources (because 1 is 25% of the total weight of 4).
+
+This allows you to prioritize certain subgroups over others in terms of resource allocation.
+In this example, queries from the ``engineering`` subgroup will be prioritized over queries
+from the ``marketing`` subgroup.
+
+.. literalinclude:: schedule-weight-example.json
+    :language: text
 
 Selector Rules
 --------------
 
-* ``user`` (optional): regex to match against user name.
+Here are the key components of selector rules in PrestoDB:
 
-* ``source`` (optional): regex to match against source string.
+* ``group`` (required): This is the name of the resource group where the query will be routed.
+
+* ``user`` (optional): This is a regular expression that matches the user who is submitting the query.
+
+* ``userGroup`` (optional): regex to match against every user group the user belongs to.
+
+* ``source`` (optional): This matches the source of the query, which is typically the
+  application submitting the query.
+
+* ``queryType`` (optional): This specifies the type of the query such as READ, WRITE, or MANAGE.
```

Comment:
I'm seeing the same too-indented formatting as above. 

first screenshot is what I'm seeing local build from the current file 
<img width="823" alt="Screenshot 2023-09-07 at 11 38 56 AM" src="https://github.com/prestodb/presto/assets/7013443/3bc12392-8595-4d91-a4a8-35e1cf5ab831">

second screenshot is built from my edited file of how I think this section should display
<img width="814" alt="Screenshot 2023-09-07 at 11 39 37 AM" src="https://github.com/prestodb/presto/assets/7013443/dc8de32b-2249-4441-886e-72e8eff3fc0e">